src/BizHawk.Emulation.Cores/Computers/DOS/DOSBox.cs

```diff
@@ -151,8 +151,12 @@ public DOSBox(CoreLoadParameters<object, SyncSettings> lp)
 				_libDOSBox.PushCDData(curDiscIndex, CDDataStruct.End, CDDataStruct.Last);
 
 				// Passing track data to the core
-				for (var trackIdx = 0; trackIdx < CDDataStruct.Last; trackIdx++) _libDOSBox.PushTrackData(curDiscIndex, trackIdx, CDDataStruct.Tracks[trackIdx]);
-			}
+				for (var trackIdx = 0; trackIdx < CDDataStruct.Last; trackIdx++)
+				{
+					// Console.WriteLine($"[CD] Adding Track {trackIdx}. Start: {CDDataStruct.Tracks[trackIdx].Start}, End: {CDDataStruct.Tracks[trackIdx].End}, End: {CDDataStruct.Tracks[trackIdx].Offset}");

```

Comment:
Not needed I guess?

Reply:
I rather keep it there, in case I need to revisit I'll have some quick debug info available